onnxruntime/core/optimizer/conv_add_fusion.cc

```diff
@@ -114,9 +120,20 @@ bool ConvAddFusion::SatisfyCondition(const Graph& graph, const Node& node) const
   }
 
   const auto& next_node = *node.OutputNodesBegin();
-  return !(!graph_utils::IsSupportedOptypeVersionAndDomain(next_node, "Add", {7}) ||
-           next_node.GetExecutionProviderType() != node.GetExecutionProviderType() ||
-           next_node.GetInputEdgesCount() != 1 || graph.IsNodeOutputsInGraphOutputs(next_node));
+  if (!graph_utils::IsSupportedOptypeVersionAndDomain(next_node, "Add", {7}) ||
+      next_node.GetExecutionProviderType() != node.GetExecutionProviderType() ||
+      next_node.GetInputEdgesCount() != 1 || graph.IsNodeOutputsInGraphOutputs(next_node)) {
+    return false;
+  }
+
+  // if (!graph_utils::IsNodeInputConstant(graph, node, 1) /* ||
+  //     (node.InputDefs().size() == 3 && !graph_utils::IsNodeInputConstant(graph, node, 2)) ||
+  //     !graph_utils::IsNodeInputConstant(graph, next_node, 1)*/
+  // ) {
+  //   return false;
+  // }
+
```

Comment:
can you delete this commented out code?

Reply:
wait - don't you need this? I'm confused. This looks like the fix, but it's commented out. The non-commented out changes here don't actually modify the logic.

Reply:
fixed